css/style.css

```diff
@@ -0,0 +1,149 @@
+html {
+  /* Setting a base font size of 10px give us easier rem calculations
+         Info: 1rem === 10px, 1.5rem === 15px, 2rem === 20px and so forth
+       */
+  font-size: 10px;
+  margin: 0;
+}
+
+body {
+  /* Native font stack https://getbootstrap.com/docs/4.2/content/reboot/#native-font-stack */
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto",
+    "Helvetica Neue", Arial, sans-serif;
+  font-size: 2rem;
+  line-height: 1.5;
+  color: #477ca7;
+  background: #333;
+  margin: 0px;
+}
+
+#main-doc {
+  display: block;
+  margin: auto;
+  border-radius: 5px;
+  background: #333;
+}
+
+#navbar {
+  width: 25%;
```

Comment:
I guess it would be good to add css rules to make your side nav bar scroll-able. You may try overflow-y: scroll

Reply:
- [ ] You are required to write at least to review comments and currently, this is the only one. Please add at least one more comment 👍